backend/src/app.js

```diff
@@ -36,7 +36,7 @@ app.use(morgan('dev'))
 // verifies user
 app.use(auth)
 
-app.use('/', routes)
+app.use('/api/', routes)
```

Comment:
Won't this make the URL to access the backend be /api/api?

Reply:
Great question—it shouldn't, specifically because right now they're of the form `http://localhost:8080/interviews/` for example if running locally. This worked great when we would have, for example, `https://my_backend_server.heroku.com/interviews` running on a different domain from the frontend server. That way we could have `https://my_backend_server.heroku.com/interviews` for the API and `https://my_frontend_server.heroku.com/interviews` for the frontend and there would be no conflict. But now, since we'll be hosting them all on one Vercel domain, we need to route the two separately, so have everything that starts with `/api` be routed to our Express functions, while everything else be routed to the frontend.